### PR TITLE
miniapps: install to bin/

### PR DESCRIPTION
**Pull request by haampie**

### CODE REVIEW

File: miniapp/CMakeLists.txt [hunk 1]
```diff
@@ -29,43 +32,50 @@ if (DLAF_BUILD_TESTING)
...
   DLAF_addPrecompiledHeaders(miniapp_triangular_solver)
+  install(TARGETS miniapp_triangular_solver RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
 endif()
 
 add_executable(miniapp_gen_to_std miniapp_gen_to_std.cpp)
 target_link_libraries(miniapp_gen_to_std PRIVATE DLAF)
 target_link_libraries(miniapp_gen_to_std PRIVATE DLAF_miniapp)
 target_add_warnings(miniapp_gen_to_std)
 DLAF_addPrecompiledHeaders(miniapp_gen_to_std)
+install(TARGETS miniapp_gen_to_std RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
```
Comment (msimberg) [inline miniapp/CMakeLists.txt:43]: Should we do:
```suggestion
DLAF_addMiniapp(miniapp_gen_to_std ...)
```
I volunteer to do this in a separate PR...

**Comment by rasolca** [comment]
Our plot scripts look for the miniapp directory.
https://github.com/eth-cscs/DLA-Future/blob/master/scripts/miniapps.py#L171
The possibility are 2:
- we should modify the scripts to take the miniapp base directory instead of the build directory
- we don't care and use our scripts only from build dirs.

**Comment by msimberg** [comment]
This:
>     * we don't care and use our scripts only from build dirs.
at least for this PR... This PR is a strict improvement since we couldn't run benchmarking scripts with installed miniapps before anyway.

**Review by rasolca — APPROVED** [review]
